packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -235,39 +245,45 @@ const createEstate = async estate => {
   );
 
   const estateId = await new Promise((resolve, reject) => {
-    estateContract.on("CreateEstate", message => {
-      const { data } = message;
-      const { args } = data;
+    const timeout = setTimeout(() => {
+      console.log(`Timeout reached`);
       estateContract.unsubscribe();
-      resolve(args._estateId);
-    });
+      action.unsubscribe();
+      reject();
+    }, EVENTS_TIMEOUT);
 
     // Some error (orphan block, failed tx) events are being triggered only from the confirmationListener
     // See more: https://github.com/tasitlabs/TasitSDK/issues/253
-    action.on("confirmation", () => {});
+    action.on("confirmation", message => {
+      const { data } = message;
+      const { confirmations } = data;
+      if (confirmations >= 1) {
+        action.unsubscribe();
```

Comment:
Should we do a `clearTimeout` here too like for the parcel one?

Reply:
On the `createEstate` function was created using "CreateEvent" as a successful event. The `confirmation` event is listening because without it, some error events not being triggered.
I can keep that way or use it as the successfull event instead of "CreateEvent" as `createParcel` does. What do you think?

Reply:
Hmm, no strong preference - I'd say `createEstate` and `createParcel` working roughly the same way probably makes sense, but I'm undecided between the two options. For now maybe just noting the difference with an in-code comment is the way to go.

Reply:
I've removed the body of the "confirmation" listener function since it's doing nothing and I think it will make that (listening only for triggering some error events) more clear. Make sense?

Reply:
Sure, that sounds good. I'll take a look at the code changes.

Reply:
Just committed.

Reply:
Yep, this looks great. Resolving this.